lib/status-bar.coffee

```diff
@@ -15,9 +15,9 @@ class StatusBar extends View
   returnFocus: null
 
   @content: ->
-    @div class: 'termination status-bar', tabindex: -1, =>
+    @div class: 'termination inline-block', tabindex: -1, =>
```

Comment:
There is no actual less/CSS class `inline-block` defined, so technically this is breaking something, it would probably work by removing it you can test this easily if by removing it would also fix issue.

If removing it fixes the issue, then it probably means this should be fixed some other way.

Reply:
@Fred-Barclay ping.

Reply:
@the-j0k3r @qub1750ul sorry I'll check this soon